### PR TITLE
Added 'SQLProvider SQL Server SSDT' to the Data recipes

### PR DESCRIPTION
**Pull request by JordanMarr**

### CODE REVIEW

**Comment by JordanMarr** [comment]
This page has been truncated to be as simple as possible (removed the extra feature added from the blog post).

**Comment by olivercoad** [comment]
It would be good to add an introduction like the other recipes.
Maybe explain what SSDT is and link to [Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/what-is-azure-data-studio?view=sql-server-ver15) or something related.

File: docs/recipes/storage/use-sqlprovider-ssdt.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
+# Using SQLProvider SQL Server SSDT
+
+## Creating a "SafeTodo" Database with Azure Data Studio
+
+### Connecting to a SQL Server Instance
+1) In the "Connections" tab, click the "New Connection" button
+
+![image](https://user-images.githubusercontent.com/1030435/111041187-26777d00-8405-11eb-9260-5d885b0ff640.png)
```
Comment (olivercoad) [inline docs/recipes/storage/use-sqlprovider-ssdt.md:8]: The images should probably be committed to the repo instead of just linking to githubusercontent

Comment (JordanMarr) [reply docs/recipes/storage/use-sqlprovider-ssdt.md:8]: The images were added using the cool new GitHub feature that lets you paste them directly into the editor. 

According to this SO post, these files do not expire and cannot be removed by the customer:
https://stackoverflow.com/questions/46374918/do-images-in-user-images-githubusercontent-com-have-expire-time

**Comment by Thorium** [comment]
Great improvement to the existing documentation! Maybe merge this as is, and then improve later further when needed?

**Comment by isaacabraham** [comment]
@JordanMarr thanks for doing this!

**Pull request merged**